### DISCUSSION TITLE
Network only?

### DISCUSSION THREAD

**Discussion by YawerSagar** [simonsobs/socs#474] [Q&A]
Hello,

I was curious if the OCS requires a network to work, or can it still be useable with only wired connections?

Thank you for your time.

**Comment by BrianJKoopman** [comment]
OCS does require a local network, since communication is all facilitated by the crossbar server which is accessed via that network. This network can be entirely wired connections.

Some details about the networking requirements can be found here: https://ocs.readthedocs.io/en/main/user/dependencies.html#networking-requirements
